src/server/data-access/submissions.ts

```diff
@@ -18,7 +19,7 @@ export async function createSubmissionsRecord(
   code: string,
   status: boolean
 ) {
-  const { client } = await createSessionClient();
+  const client = (await serviceClient.createSession()).client;
```

Comment:
as we are working on the abstraction, can we also abstract out the database object part as well

`const { databases } = await serviceClient.createSession()`